tests/plots/test_pinball_and_exceedance.py

```diff
@@ -0,0 +1,25 @@
+from plots.make_pinball_and_exceedance_plots import make_pinball_or_exceedance_plot
+
+from datetime import datetime
+
+
+def test_make_pinball_plot(db_session, metrics_pinball):
```

Comment:
just a question, but for these tests, do you not need to make any assertions?

Reply:
its a good question of how to test making plots?